### PR TITLE
Add windowrules for noblur and noshadow

### LINKED ISSUE #883: Windowrules for disabling shadows & borders
basically just

```
windowrule=noshadows,...
windowrule=noborders,...
```

mostly for shimeji (spamtong my beloved)

image
image

### PR DESCRIPTION
**Pull request by ThatOneCalculator**
#### Describe your PR, what does it fix/add?

Closes #883

### CODE REVIEW

File: src/render/OpenGL.cpp [hunk 1]
```diff
@@ -1083,7 +1083,7 @@ void CHyprOpenGLImpl::renderRoundedShadow(wlr_box* box, int round, int range, fl
     RASSERT((box->width > 0 && box->height > 0), "Tried to render shadow with width/height < 0!");
     RASSERT(m_pCurrentWindow, "Tried to render shadow without a window!");
 
-    if (!pixman_region32_not_empty(m_RenderData.pDamage))
+    if (!pixman_region32_not_empty(m_RenderData.pDamage) || (m_pCurrentWindow && m_pCurrentWindow->m_sAdditionalConfigData.forceNoShadow))
```
Comment (vaxerski) [inline src/render/OpenGL.cpp:1086]: it's better to nope earlier - in the decoration class itself. see CHyprDropShadowDecoration::render

File: docs/Hyprland.1 [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .\" Automatically generated by Pandoc 2.5
 .\"
-.TH "Hyprland" "1" "11 Oct 2022" "" "Hyprland User Manual"
+.TH "Hyprland" "1" "20 Oct 2022" "" "Hyprland User Manual"
```
Comment (vaxerski) [inline docs/Hyprland.1:3]: I believe this is accidental

Comment (ThatOneCalculator) [reply docs/Hyprland.1:3]: Yea, github actions :(

File: src/render/OpenGL.cpp (hunk 1 above)
Comment (ThatOneCalculator) [reply src/render/OpenGL.cpp:1086]: There's no `::render` that I can find, but there's `::draw`. Should I go for draw?

Comment (vaxerski) [reply src/render/OpenGL.cpp:1086]: yeah I was paraphrasing

**Pull request merged**